testcases/client-server/dnf-FTP/server

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+mkdir -p /run/fedora-networking
+/usr/bin/cp -u dummy.spec /run/fedora-networking/
+cd /run/fedora-networking
+
+rpmbuild -ba dummy.spec
```

Comment:
can we use something like: 
~~~
rpmbuild --define "_topdir /var/run/fedora-networking" -ba dummy.spec
~~~
To keep ${HOME} clean?

Reply:
Moved.